services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -348,6 +348,7 @@ public List<Integer> layersList() {
    * {@link DirectionsCriteria#ANNOTATION_CONGESTION_NUMERIC}
    * {@link DirectionsCriteria#ANNOTATION_MAXSPEED}
    * {@link DirectionsCriteria#ANNOTATION_CLOSURE}
+   * {@link DirectionsCriteria#ANNOTATION_TRAFFIC_TENDENCY}
```

Comment:
docs also has to be added to 
https://github.com/mapbox/mapbox-java/blob/a15e481b435656a46039be3b128d90323a6005e0/services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java#L361-L379
,
https://github.com/mapbox/mapbox-java/blob/a15e481b435656a46039be3b128d90323a6005e0/services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java#L1402-L1403
, and
https://github.com/mapbox/mapbox-java/blob/a15e481b435656a46039be3b128d90323a6005e0/services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java#L1423-L1424